.github/workflows/publish.yml

```diff
@@ -27,6 +27,11 @@ jobs:
             ghcr.io/supabase/gotrue
             436098097459.dkr.ecr.us-east-1.amazonaws.com/gotrue
             646182064048.dkr.ecr.us-east-1.amazonaws.com/gotrue
+            supabase/auth
+            public.ecr.aws/supabase/auth
```

Comment:
We should remember to [update CLI] (https://github.com/supabase/cli/blob/04122dde5c63678a0d657d9c6c6f70931a985fcb/internal/utils/misc.go#L39) to pull from this in the future

Reply:
deployment scripts that reference the docker image will need to be updated as well